pkg/util/logging/default_logger.go

```diff
@@ -19,9 +19,12 @@ package logging
 import (
```

Comment:
Let's move all of the changes currently in this file that are related to creating the new flag to a new file, `format_flag.go`, in this same package.